docs/topics/proc_deploying-an-application-to-openshift.adoc

```diff
@@ -72,6 +81,7 @@ The route information of a pod gives you the base URL which you use to access it
 +
 [source,bash,options="nowrap",subs="attributes+"]
 ----
-$ curl http://{app-name}-{project-name}.{os-route-hostname}
-{"content":"Greetings!"}
+$ curl {parameter-url}
+{parameter-response}
```

Comment:
The gist of RHOARDOC-1233

Reply:
Will this conflict with https://github.com/fabric8-launcher/launcher-documentation/pull/1087?

Reply:
Agreed, thanks :-)